### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by jodavid444**
# In This Pair Programming Project:

- we have inserted a huge amount of data to our local database machine.
- Then we tested the performance of our database query if it is more than one second
- Lastly we take an optimization methods;
---
#### Our Optimization methods:-
- creating an index to the table for the searchable column
## QUERY ONE

214057722-61bc6a50-d426-4d56-9220-b070222ee2ec (1)

214057742-18dde570-3d64-4cb5-b83d-3d74ded926a6

## QUERY TWO

query_two

query-two-ii

## QUERY THREE 

query-three

query-three-ii

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. 

image
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -22,6 +22,9 @@ CREATE TABLE owners (
   PRIMARY KEY (id)
 );
 
+-- updating the schema of owner table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
 CREATE TABLE species (
   id INT GENERATED ALWAYS AS IDENTITY,
```
Comment (omar-muhamad) [inline schema.sql]: - [ ] Please add the solution you used to decrease the execution time for your queries.
For example, if you added indexes for FOREIGN KEYS(the best way to solve the performance problems) Kindly add them here in the schema file.

File: README.md [hunk 2]
```diff
@@ -85,8 +83,6 @@ Install this project with:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Joseph David**
```
Comment (omar-muhamad) [inline README.md:86]: - [ ] Kindly also add your collaborator @juaryimami name and profile links to the Authors section on the Readme file.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @jodavid444 and @juaryimami,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by jodavid444** [comment]
Thanks for the review I have Implemented the required changes kindly review it again @rloterh

File: README.md (hunk 2 above)
Comment (jodavid444) [reply README.md:86]: Thanks, for the review I have implemented the required changes kindly review it again @Omar-Muhamad

File: schema.sql (hunk 1 above)
Comment (jodavid444) [reply schema.sql]: Thanks for the review I have added the solution used to decrease the execution time of the queries, kindly review it again @Omar-Muhamad

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @jodavid444, and @juaryimami👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Well done with the performance audit. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**